src/server/global_notifier.hpp

```diff
@@ -21,13 +21,11 @@
 
 #include "impl/collection_notifier.hpp"
 #include "shared_realm.hpp"
+#include "realm/object_id.hpp"
```

Comment:
This is a core header and not an object store header. It also isn't needed here as it can just be forward-declared.